src/main/resources/com/whiteclarkegroup/liquibaselinter/config/lqlint-implicit.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I can pull this out, but I thought it would be valuable. It provides a mechanism where `liquibase-linter` could enable some core default rules and reporters by default. Right now there aren't any default rules, but there could be. I think the Java code itself to do what you suggest is going to be more than what exists with this solution (but not significant).

I'll remove it, but I'll probably make another issue to introduce this feature of default rules and reporters. I think having this centralized file would increase visibility to all the implicit/default configuration going on in the `liquibase-linter`.

Reply:
The bulk of the changes in `ConfigLoader` relate to refactoring the feature that merges rules into one that can also merge reporters. To remove this feature only removes this from `ConfigLoader` (and this `lqlint-implicit.json` file):
```
                if (config != null) {
                    return loadImports(resourceAccessor, config, singletonList(LQLINT_CONFIG_IMPLICIT_PATH));
                }
```
And replaces it with these new lines in `LintAwareChangeLogParser`:
```
        if (!config.getReporting().containsKey(ConsoleReporter.NAME)) {
            new ConsoleReporter(ReporterConfig.builder().build());
        }
```

Have I convinced you that this could be a valuable feature worth keeping?

Reply:
In terms of:

> a mechanism where liquibase-linter could enable some core default rules and reporters by default. Right now there aren't any default rules, but there could be

Agree on the case for some sensible default/recommend config, we're going to do this on an opt-in basis under #65 - would be interested to see your thoughts there on what would be in/out of a recommended config (for both rules and reporters).

In the meantime I think defaulting in the code rather than a noop file is the way to go.